src/compiler/scala/tools/nsc/symtab/SymbolLoaders.scala

```diff
@@ -25,7 +26,10 @@ abstract class SymbolLoaders {
   val platform: backend.Platform {
     val symbolTable: SymbolLoaders.this.symbolTable.type
   }
+
   import symbolTable._
+  val statistics: Statistics with ReflectStats
```

Comment:
This is required because of a clash of path dependent types betwen `SymbolTable` and `Global`, IIRC.

Reply:
Would be nice to get rid of it if possible.. If you don't manage to convince the compiler that two paths are the same, you could also do a cast.

Reply:
I can do a cast. I thought this solution would be nicer.

Reply:
Oh no, I remember now why this was done. `statistics` in `symbolTable` is `protected`, so it's not accessible here. That's why the trait that mixes in `SymbolLoaders` and `SymbolTable` defines `statistics` using the knowledge from the inherited `SymbolTable`. As far as I see it, there's no other solution than this because abstract members cannot be `private[pkg]`.

Reply:
Let's make `statistics` public then - or is there a good reason to keep it protected?

Reply:
It's protected so that this addition is binary compatible, otherwise it wouldn't because it leaks to public APIs. Unless you see a way to make bincompat, I don't see another solution.

Reply:
`scala.reflect.internal` is excluded from binary compatibility (https://github.com/scala/scala/blob/2.12.x/src/reflect/mima-filters/2.12.0.forwards.excludes#L1). In any case, Scala's `protected` translates to `public` in bytecode.

Reply:
My first iteration made it public and I got a MiMa error somewhere in a public interface, but I'll try again to double-check -- I may have done something wrong.